### PR TITLE
recreating a0viedo's changes

### PR DESCRIPTION
**Pull request by Sequoia**
getting @a0viedo's FAQ in from https://github.com/nodeschool/discussions/pull/861, which got stale before we merged it. Thx @a0viedo!

### CODE REVIEW

**Comment by Sequoia** [comment]
:+1: \#1

**Comment by SomeoneWeird** [comment]
You could have just rebased :)

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### I installed Node.js, now what?
+
+**Windows** - Open the Start Menu and look for an icon that says "Node.js Command Prompt". Once you open it, you could for example type `npm install javascripting -g` to install Javascripting. If you want to install other workshopper, use the same command replacing the module name.
+
+**Linux/OSX** - Open a terminal and type `npm install javascripting -g` to install Javascripting. If you want to install other workshopper, use the same command replacing the module name.
+
+### I installed Node.js but when I type `node` in my terminal it says `'node' is not recognized` (or `node: command not found`)
+
+It's likely that you have a problem with your installation. Go through the installation steps again and if you still have the problem, open an issue telling us exactly which steps did you followed to install Node.
```
Comment (SomeoneWeird) [inline FAQ.md]: s/did//

**Comment by Sequoia** [comment]
@SomeoneWeird tried, didn't feel like sorting it out :yum: 
```diff
[![Gitter][gitter-image]][gitter-chat]

discussions
===========

Need help? or just wanna ask a question? [open an issue][discussion/issues] on this repo!

NodeSchool organizers can have discussions [over here][organizers]. If you want to create your own NodeSchool workshop, have a look at the [workshoppers repo][workshoppers].

**Disclaimer**: In you open an issue, it's content, title and code formatting may be edited by the repository owners to improve overall understanding of the problem.

# FAQs

## Installation

### How do I know that Node.js installed correctly?

**Windows** - Open the Start Menu and look for an icon that says "Node.js Command Prompt". Type `node` in the terminal and press Enter. If a weird greater symbol (>) appeared in your terminal with a blinking underscore, then you're able to run Node.

**Linux/OSX** - Open a terminal and type `node`. If a weird greater symbol (>) appeared in your terminal with a blinking underscore, then you're able to run Node.

### I installed Node.js, now what?

**Windows** - Open the Start Menu and look for an icon that says "Node.js Command Prompt". Once you open it, you could for example type `npm install javascripting -g` to install Javascripting. If you want to install other workshopper, use the same command replacing the module name.

**Linux/OSX** - Open a terminal and type `npm install javascripting -g` to install Javascripting. If you want to install other workshopper, use the same command replacing the module name.

### I installed Node.js but when I type `node` in my terminal it says `'node' is not recognized` (or `node: command not found`)

It's likely that you have a problem with your installation. Go through the installation steps again and if you still have the problem, open an issue telling us exactly which steps did you followed to install Node.

### I have Node.js installed but when I type `javascripting` in my terminal it shows an error

If you know that Node installed [correctly][node-installed-correctly], then you may have a problem with the installation of the workshopper. Try to reinstall the workshopper. After that, if you still have the problem open an issue with your error and tell us exactly what happened.

## Working on challenges

### I can't find what is wrong!

Don't panic. Open an issue, give us your code, the workshopper name, the challenge name and be patient. Someone will try to help you soon.

### The challenge is wrong!

All the workshoppers are Open Source. If you know how to fix it, just submit a pull request to the workshopper's repository explaining the problem. If you don't know how to fix it, open an issue with the title `Improvement - Workshopper name, challenge name` and all the aditional info you consider it's neccessary in it's content.



<sub>
<b>TOC</b><br/>
<b><a href="#when-you-have-a-problem">When you have a problem</a><br/></b>
&nbsp;<b><a href="#in-general">In general</a>:</b> <a href="#1---be-nice">1 - Be nice</a>, <a href="#2---be-public">2 - Be public</a>, <a href="#3---be-patient">3 - Be patient</a>, <a href="#4---non-english-is-okay">4 - Non-English is okay</a><br/>
&nbsp;<b><a href="#suggestions">Suggestions</a>:</b> <a href="#check-your-installation">Check your installation</a>, <a href="#provide-full-command-output">Provide full command output</a>, <a href="#use-markdown">Use markdown</a>, <a href="#say-thanks">Say thanks!</a>, <a href="#close-your-issue">Close your issue</a>, <a href="#bikeshedding">Bikeshedding</a><br/>
<b><a href="#about-labels">About Labels</a><br/></b>
<b><a href="#answering-issues">Answering Issues</a>:</b> <a href="#use-links">Use links</a>, <a href="#end-with-a-question">End with a question</a>, <a href="#reformat-issues">Reformat issues</a>, <a href="#apply-priorities">Apply priorities</a>, <a href="#close-issues">Close issues</a>
</sub>

# When you have a problem:

## In general

### 1 - Be nice

All of the workshops and all of the discussions are done by volunteers and there is no guarantee that all questions get answered. If you only provide negative feedback and not constructive criticism your issue will probably get closed with no response.

### 2 - Be public

Don't email workshop authors directly, use the public discussion forum (the Issues) for all communication, that way your question and any answers you get can benefit everyone else.

### 3 - Be patient

It may take a day or two for people to respond here. Please don't send multiple messages in a row, it won't help your question get answered any faster.

### 4 - Non-English is okay!

If you can't read or write english well its better to write additionally your question in your native language. There are many people on this site and maybe people can help you in your mother-tongue.

## Suggestions

### Check your installation

Often the installation on [windows](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+label%3Awindows+) makes problems. But also other systems can experience troubles. Please also check out [the installation suggestions](https://github.com/nodeschool/discussions/wiki/Installation-suggestions) if you have problems.

### Provide *full* command output

To help us debug your problem please include all output from the commands you ran, including any error messages you received.

It also helps to include your node, npm, git or other CLI command versions, which you can get by running `node -v`, `npm -v` etc.

<<<<<<< HEAD
### Use markdown

Github offers a lot of support to [format your issue](https://help.github.com/articles/github-flavored-markdown/). It makes it easier for other people to understand your problem.

### Say thanks!

The person that helped you did it out of the kindness of her/his heart. Tell your appreciation by saying thanks :smile: !

### Close your issue

If you don't close your issue another person will have to clean it up later.

### Bikeshedding

Its okay to [bikeshed](http://en.wiktionary.org/wiki/bikeshedding) issues but if you do it would be nice to update it every 14days or so to let people know you still have interest.

# About Labels

Only [owners](https://github.com/orgs/nodeschool/teams/owners) can add labels to the dicussions issues. We added a lot of labels that makes it easier for contributors to understand the content of an issue. Here is their intended meaning:

| Label | Meaning |
| ----- | ------- |
| [bikeshedding](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aopen+label%3Abikeshedding) | This issue is kept open in case someone wants to jump in. It should be updated every now and then (~14days). |
| [contains hints for improvements](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aopen+label%3Acontains-hint-for-improvement) | This contains hints on how a workshopper/tutorial/website etc. can be improved. The issue should be closed when the improvement was added to the content in question as a [PR](https://help.github.com/articles/using-pull-requests/) or issue. |
| [general node.js](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aopen+label%3Ageneral-node.js) | This is a question not specific to a content in this repository but rather to node.js in general. |
| [installation](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aopen+label%3Ainstallation) | This is an issue with the installation of node or a workshopper. |
| [mac](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aopen+label%3Amac) | This is an issue specific to computers with Mac OS X. |
| [needs some love](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-some-love+) | This issue is not simple to answer or process. More effort than average needs to be put into this issue and it needs the love and support of the greater community. | 
| [probably self resolved](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aclosed+label%3Aprobably-self-resolved) | This label is for the case that the person that asked the question has not answered for a longer period of time (~14 days). It should be used by the person who closes an inactive thread to show that she/he thinks that the user probably resolved this issue himself. Usually only closed issues should have this label. |
| [seems resolved](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aclosed+label%3Aseems-resolved) | This label is to indicate that the issue seems resolved but the user forgot to close the issue. Usually only closed issues should have this label. |
| [question](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+label%3Aquestion+is%3Aopen) | This issue contains a question that is not a problem. |
| [waiting for feedback](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aopen+label%3Awaiting-for-feedback) | This label is to remember that the issue has been answered but is waiting on additional information or more response. |

If you want to add labels to issues, don't be afraid to ask for permissions. Its a github restriction that only owners can add labels.

If you have an issue that doesn't match a label, feel free to add a label and describe it here.

# Answering Issues

Owner or not: If you have an answer to a question please don't hesitate to add a comment. There are people who answer issues regularily but those are also volunteers and also only try to do their best.

It is not necessary to follow this but there are some things that we found useful to keep in mind:

## Use links

Many questions are similar. If you can link to other issues or web content that makes your life easier. Keep in mind that the other person still needs an explanation why you give this link.

## End with a question

If you add a question at the end it puts the ball back to the inital questionee and indicates that you think that this comment should fix the issue of the person. i.e.:

 - Could you try that?
 - Does that help you?
 - Does that solve your problem?
 - Do you need more information?

## Reformat issues

Owners can edit issues. It is often not a good idea to change the issues but adding [Syntax Highlighting](https://help.github.com/articles/github-flavored-markdown/#syntax-highlighting) often helps readability and increases the chances of a good response so that we consider that a good exception.

## Apply priorities

In case you feel confident to answer multiple issues it might be a good idea to follow priorities. An example for priorities might be:

1. [Issues with no labels](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aopen++labels+): Add labels
2. [Issues with no comments](https://github.com/nodeschool/discussions/issues?q=is%3Aopen+is%3Aissue+comments%3A0+): Think of an answer
3. [Issues that are least recently updated](https://github.com/nodeschool/discussions/issues?q=is%3Aissue+is%3Aopen+no%3Alabels): Add the `waiting-for-response` label in case its not given and ask a question in case none has been asked.
4. [Issues that `need-some-love`](https://github.com/nodeschool/discussions/issues?q=is%3Aopen+is%3Aissue+label%3Aneeds-some-love): Try to think how to resolve those because they need your love!

## Close issues

If you see an old issue lurking around feel free to apply labels as mentioned above and **close the issue if reasonable**!
=======

[gitter-image]: https://badges.gitter.im/Join%20Chat.svg
[gitter-chat]: https://gitter.im/nodeschool/discussions?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
[discussion/issues]: https://github.com/nodeschool/discussions/issues
[organizers]: https://github.com/nodeschool/organizers
[workshoppers]: https://github.com/nodeschool/workshoppers
[node-installed-correctly]: https://github.com/nodeschool/discussions#how-do-i-know-that-nodejs-installed-correctly
>>>>>>> added faq section to the readme
```

**Comment by a0viedo** [comment]
my bad, should have been merged back then. Thank you @Sequoia!

**Comment by Sequoia** [comment]
@SomeoneWeird re: that fixup, please feel free to patch-- should flow into this PR automatically.

**Comment by Sequoia** [comment]
@a0viedo np! We were the ones who held you up :smile_cat:. There's no official workflow it's sorta confusing... I propose 2 thumbs & hit that green button :100:

**Comment by martinheidegger** [comment]
:+1:

**Pull request merged**